src/main/webapp/app/tasks/default-task.service.ts

```diff
@@ -20,6 +20,16 @@ export class DefaultTaskService implements TaskService {
     return this.http.delete<void>(this.baseUrl + '/tasks/' + id);
   }
 
+  done(id: string): Observable<void> {
+    // return this.http.done<void>(this.baseUrl + '/tasks/' + id);
```

Comment:
This comment is non-working code. Angular's HttpClient does not provide a _done_ method. The methods available relate to the different HTTP calls: GET, POST, PUT, DELETE, ...